lms/services/course.py

```diff
@@ -142,47 +143,55 @@ def search(  # noqa: PLR0913, PLR0917
     def get_courses(
         self,
         instructor_h_userid: str | None = None,
-        organization: Organization | None = None,
+        admin_organization_ids: list[int] | None = None,
```

Comment:
We might eventually add another `organization_ids` parameter if we ever include the organizations concept in the dashboard, or example as a dropdown filter.

The situation is the same for h_userids (filtering by this users) vs  instructor_h_userid (only courses where that user has access).